action-scripts/default.py

```diff
@@ -1,3 +1,7 @@
 from default_scripts import ignore_filters
 
-ignore_filters.main()
+def main():
```

Comment:
when do you want to roll out the new bpy-addon-build, since the args will change for that (needing to pass in context). Could do in a small follow up PR

Reply:
I think I'll roll it out after this PR. I don't want to drag this PR on for too long, and BpyBuild still needs a little more time before the API is released